### DISCUSSION TITLE
End-user feedback

### DISCUSSION THREAD

**Discussion by qcaas-nhs-sjt** [lsc-sde/lsc-sde#68] [Q&A]
As I understand it we have people using the workspaces for a variety of purposes. I think that it would be really useful if we could get a process together for users of the system to provide some meaningful feedback such as:

* Additional functionality that would benefit the end-user
* Improvements to useability of the system
* Get feedback on what we are doing
* Ideas and innovations on direction
* etc

Once we've got this feedback we can then use this to input into the roadmap and this should also increase our visibility to the community. 

So how can we best get users engaged and get them providing this feedback to us?

**Comment by qcaas-nhs-sjt** [comment]
@vvcb @m1p1h 
Ideas on a postcard please

**Comment by m1p1h** [comment]
There's a few things we could do at LU to help develop requirements e.g.:

- Establish a dedicated group of researchers and data scientists at LU who already work with or are looking to access NHS data
- Jo/Nicola could use their HIC channels to promote LSCSDE early adopter access. I could put a request out to DSI looking for users
- Run some workshops/ one-on-one interviews to gather feedback/requirements with the group or individual researchers

The other thing we probably need soon is user guides and walkthrough videos to increase engagement.

**Comment by vvcb** [comment]
Agree regarding the need for this. Will pick up offline to ensure we are aligned with wider work and not duplicating effort.
